switchapi/switch_pd_port.c

```diff
@@ -1,21 +1,29 @@
 /*
- * Copyright 2013-present Barefoot Networks, Inc.
- * Copyright (c) 2021-2022 Intel Corporation.
- * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2022 Intel Corporation.
```

Comment:
This is a derivative of an existing work, and should continue to carry the Barefoot Networks copyright.

You could conceivably replace "present" with the year when Barefoot officially became part of Intel, since the end date is supposed to represent the last year in which the named author contributed intellectual property to the document. (That being said, I doubt anyone would care. Feel free to consider this a nit not work being picked.)

I believe (but am not in a position to state authoritatively) that we should include the SPDK license identifier even if we use the longer copyright message. The SPDX identifier facilitates scanning files for licensing information.